specification/loadtestservice/LoadTestService/client.tsp

```diff
@@ -57,7 +57,8 @@ interface TestRunOperations {
   listMetrics is LoadTestRun.listMetrics;
   listTestRuns is LoadTestRun.listTestRuns;
   #suppress "@azure-tools/typespec-azure-core/no-operation-id"
-  stopTestRun is LoadTestRun.stop;
+  @clientName("stopTestProfileRun", "javascript,python,java,csharp,go")
```

Comment:
@tadelesh Is there any shortcut if I could change the client name for all languages?

Reply:
no for now, we are considering to add support like `~typescript`.

Reply:
okay, btw it seems the ~typescript may not help there.